test/common/testlib.py

```diff
@@ -271,9 +299,18 @@ def eval_js(self, code, no_trace=False):
         return None
 
     def call_js_func(self, func, *args):
+        """Call a JavaScript function
+
+        :param func: JavaScript function to call
+        :param args: arguments for the JavaScript function
+        """
```

Comment:
This is rather stub-y -- this does not really explain much beyond the signature. But we can build on this :-)